config/Dockerfiles/linchpin/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM fedora:latest
+LABEL name="contrainfra/linchpin-executor" \
+      maintainer="https://github.com/CentOS-PaaS-SIG/linchpin" \
+      version="0.0.5" \
+      description="LinchPin executable container running Fedora 28"
```

Comment:
should we just say latest fedora here since the FROM is fedora:latest?

Reply:
There is an argument that [containers shouldn't be tagged to "latest"](https://developers.redhat.com/blog/2016/02/24/10-things-to-avoid-in-docker-containers/)

Reply:
Yeah. I plan to change that to the latest current version of Fedora. In this case that should say `fedora:28` in the FROM section. I can update the description here. 

I also wanted to change the name to just linchpin (or linchpin-container), but wanted to have a discussion about how this would be the upstream container image, which others could build upon. I'm not sure of the good/bad of doing that so I wanted opinions.